### ISSUE TITLE
`Mongotail EXCEPTION - Unknown command operation` when list collection indexes

### ISSUE THREAD

**Issue by mrsarm** [mrsarm/mongotail#15]
Listing indexes from a collection with this: `db.foo.getIndexKeys()`, Mongotail gets this:

```
Mongotail EXCEPTION - Unknown command operation
Dump: {"ns": "test.foo", "command": {"listIndexes": "foo"}, "ts": ISODate("2016-09-29T12:39:39.280Z"), "op": "command"}
Mongotail EXCEPTION - Unknown registry
Dump: {"ns": "test.foo", "command": {"listIndexes": "foo"}, "ts": ISODate("2016-09-29T12:39:39.280Z"), "op": "command"}
```

**Comment by mrsarm** [comment]
Same error with exactly the same output calling `getIndexSpecs()`, `getIndexes()` or `getIndices()`.

**Comment by mrsarm** [comment]
Released in version 2.1.0
